setup.cfg

```diff
@@ -22,11 +22,22 @@ ignore =
 	# (check https://github.com/wemake-services/wemake-python-styleguide/issues/1441)
 	WPS436,
 	# Our private objects are fine to import
-	WPS450 
+	WPS450,
+	# We need multine loops
+	WPS352,
+	# We love f-strings
+	WPS305,
+	# There are no bad quotes
+	Q000
 	
 per-file-ignores =
-	# Unused modules are allowed in __init__, to reduce imports
-	__init__.py: F401
+	__init__.py:
+		# Unused modules are allowed in __init__, to reduce imports
+		F401,
+		# No docstrings are allowed in __init__
+		D104
```

Comment:
Si pide solo docstring para el módulo, tampoco lo veo mal.

Reply:
Our __init__.py files currently do not have docstrings and they would add little information. But we could indeed add them.

Reply:
Actually, we do have some with docstrings. This needs to change.